cmd/eudico/subnet.go

```diff
@@ -100,27 +100,31 @@ var addCmd = &cli.Command{
 			Name:  "parent",
 			Usage: "specify the ID of the parent subnet from which to add",
 		},
-		&cli.Uint64Flag{
+		&cli.StringFlag{
 			Name:  "consensus",
-			Usage: "specify consensus for the subnet (0=delegated, 1=PoW, 2=Tendermint, 3=Mir)",
+			Usage: "specify consensus for the subnet (Delegated, PoW, Tendermint, Mir)",
 		},
 		&cli.IntFlag{
-			Name:  "checkperiod",
-			Usage: "optionally specify checkpointing period for subnet (default = 10epochs)",
+			Name:  "check-period",
+			Usage: "optionally specify checkpointing period for subnet (default = 10 epochs)",
 		},
 		&cli.StringFlag{
 			Name:  "name",
 			Usage: "specify name for the subnet",
 		},
 		&cli.StringFlag{
-			Name:  "delegminer",
+			Name:  "delegated-miner",
 			Usage: "optionally specify miner for delegated consensus",
 		},
 		&cli.Uint64Flag{
 			Name:  "min-validators",
 			Usage: "optionally specify number of validators in subnet",
 			Value: 0,
 		},
+		&cli.IntFlag{
```

Comment:
Ideally, I would say that even if we make this an optional argument, we should offer a reference value for each consensus algorithm implementation (or at least a lower bound, as with every consensus algorithm the required finality may be different)

Reply:
Do we know the reference values for each consensus now? If we do I can add them. Probably we should do the same with `checkPeriod`?

Reply:
Not yet, probably worth adding a `FIXME` for future work. We could come up with something for the current consensus algorithms but it'll probably not be accurate.

Good idea also for the `checkPeriod'

Reply:
Fixed